receiver/splunkenterprisereceiver/config.go

```diff
@@ -2,3 +2,65 @@
 // SPDX-License-Identifier: Apache-2.0
 
 package splunkenterprisereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkenterprisereceiver"
+
+import (
+	"errors"
+	"net/url"
+	"time"
+
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+	"go.uber.org/multierr"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkenterprisereceiver/internal/metadata"
+)
+
+var (
+	errBadOrMissingEndpoint = errors.New("Missing a valid endpoint")
+	errMissingUsername      = errors.New("Missing valid username")
+	errMissingPassword      = errors.New("Missing valid password")
+	errBadScheme            = errors.New("Endpoint scheme must be either http or https")
```

Comment:
Go styleguides requires error messages start with a lower case since it is more than likely going to be added to a log line, like:

```golang
fmt.Println("An error occured: ", err)
```